x-pack/plugins/security_solution/public/siem_migrations/rules/service/rule_migrations_service.ts

```diff
@@ -62,26 +73,39 @@ export class SiemRulesMigrationsService {
         // send notifications for finished migrations
         pendingMigrationIds.forEach((pendingMigrationId) => {
           const migration = results.find((item) => item.id === pendingMigrationId);
-          if (migration && migration.status === 'finished') {
+          if (migration?.status === 'finished') {
```

Comment:
Could we have any enum like `MigrationStatus` ? and use it like `MigrationStatus.LOADING`,  `MigrationStatus.RUNNING`